tribool.go

```diff
@@ -0,0 +1,13 @@
+package bff
+
+type Tribool *bool
```

Comment:
Nit: I think "Tristate" explains this better than "Tribool"

```suggestion
type Tristate *bool
```

Rename as appropriate elsewhere.